_rules/form-control-accessible-name-e086e5.md

```diff
@@ -210,15 +210,15 @@ Hidden to assistive technologies.
 Role has [explicitly](#explicit-role) been set to something that isn't a form field.
 
 ```html
-<input role="presentation" />
+<input role="banner" />
 ```
 
 #### Inapplicable Example 4
 
 Option inherits from input, but has a required context role of listbox which inherits from select. We should therefore not consider option as applicable.
 
 ```html
-<select role="none">
+<select>
```

Comment:
This will cause the example to fail rather than be inapplicable.

Reply:
Hmm, yes, it now fails on the `select` 🙄

Reply:
Hmm, yes, it now fails on the `select` 🙄